patches/server/0709-Line-Of-Sight-Changes.patch

```diff
@@ -32,15 +34,15 @@ index 2247d1c7c7dd1c225c511b44d6aeb5c3e75bdb2a..1cc8a85ae69c98987ab516418808b688
 +        if (from.getWorld() != to.getWorld()) return false;
 +        Vec3 vec3d = new Vec3(from.getX(), from.getY(), from.getZ());
 +        Vec3 vec3d1 = new Vec3(to.getX(), to.getY(), to.getZ());
-+        if (vec3d1.distanceTo(vec3d) > 128.0D) return false;
++        if (vec3d1.distanceToSqr(vec3d) > 16384.0D) return false; //Return early if the distance is greater than 128 blocks
```

Comment:
I'd prefer we use `128D * 128D` so it's clear what it's a square of.

Reply:
Done!